chia/util/api_decorators.py

```diff
@@ -59,3 +59,14 @@ def inner():
         return func
 
     return inner()
+
+
+def reply_type(type):
+    def wrap(func):
+        def inner():
+            setattr(func, "reply_type", type)
```

Comment:
is this attribute used anywhere?

Reply:
Almog uses this in #8093 - in that patch, he is building the expected message return types into attributes on the sending functions.

Reply:
I see